Libraries/Opc.Ua.Server/Diagnostics/CustomNodeManager.cs

```diff
@@ -4777,7 +4758,6 @@ protected NodeState AddNodeToComponentCache(ISystemContext context, NodeHandle h
         private ServerSystemContext m_systemContext;
         private string[] m_namespaceUris;
         private ushort[] m_namespaceIndexes;
-        private Dictionary<uint, IDataChangeMonitoredItem> m_monitoredItems;
```

Comment:
I have to admit I do not fully understand this change. It seems the m_moniteredItems was not used at all, is that correct? Do we know that people who have extended/derived from CustomNodeManager2 did not use it?

I'm also wondering: the method CreateMonitoredItems is implemented in several node managers, with always slightly different implementations. Is this by intention? (CustomNodeManager2, CoreNodeManager and SampleNodeManager)

Reply:
I think the idea is to not enforce specific implementations and how the monitored items handles are implemented in a node manager.